server/src-lib/Hasura/Eventing/HTTP.hs

```diff
@@ -170,7 +170,17 @@ data HTTPRespExtra (a :: TriggerTypes)
   , _hreContext  :: Maybe ExtraLogContext
   }
 
-$(J.deriveToJSON (J.aesonDrop 4 J.snakeCase){J.omitNothingFields=True} ''HTTPRespExtra)
+instance J.ToJSON (HTTPRespExtra 'Scheduled) where
```

Comment:
Is it possible to define this as `instance J.ToJSON (HTTPRespExtra a) where` to avoid duplicating ?